### PR TITLE
Add check examples for PWR0{28,29,30}

### PR DESCRIPTION
**Pull request by roberteg16**

### CODE REVIEW

File: PWR030/example.c [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    float *bb = b + k * size;
+    for (int i = 0; i < m; ++i) {
+      const float *aaa = aa + i * o;
+      float *bbb = bb + i;
+      for (int j = 0; j < o; ++j) {
+        bbb[j * m] = aaa[j];
+      }
+    }
+  }
+}
```
Comment (jgonzac) [inline PWR030/example.c]:
```suggestion
void example(float *a, float *b, unsigned n) {
  for (unsigned i = 0; i < n; i++) {
    a = b + i * n;
    for (unsigned j = 0; j < n; j++) {
      a[j * n] = 0;
    }
  }
}
```

Comment (roberteg16) [reply PWR030/example.c]: Much better :laughing: 

Done!

**Review by javinovo — APPROVED** [review]
I kind of miss some very very brief comment clarifying that `n` is the size (or maybe name it `size` instead?). Also in one example `b` is the output and `a` the input and in another it is the other way around

**Comment by roberteg16** [comment]
> I kind of miss some very very brief comment clarifying that n is the size (or maybe name it size instead?). Also in one example b is the output and a the input and in another it is the other way around

Named `size` and now unified which ones are the outputs and which the inputs.

**Pull request merged**